web/index.php

```diff
@@ -64,10 +65,14 @@
     $app['config']['cdbxml_proxy']['enabled']) {
     $app->before(
         function (Request $request, Application $app) {
-            /** @var \CultuurNet\UDB3\Http\Proxy\CdbXmlProxy $cdbXmlProxy */
+            $psr7Request = (new DiactorosFactory())->createRequest($request);
+            /** @var \CultuurNet\UDB3\Http\Proxy\Proxy $cdbXmlProxy */
             $cdbXmlProxy = $app['cdbxml_proxy'];
 
-            return $cdbXmlProxy->handle($request);
+            $psr7Response = $cdbXmlProxy->handle($psr7Request);
```

Comment:
I'm pretty sure this cdbXml part can be removed. But maybe this should be done in a separate PR?